CHANGELOG.md

```diff
@@ -1,11 +1,11 @@
-# Changelog
-All notable changes to this project will be documented in this file.
+# Change log
```

Comment:
Is changeling being split into two words intentional? Maybe I just haven't noticed this before 😭😭😭

Reply:
LOL. Same exact thought.